docs/ocean/test_cases/isomip_plus_at_lanl.rst

```diff
@@ -237,13 +237,13 @@ Take a look at which branch were on:
 
    git logg
 
-We don't start off on ``MPAS-Dev/MPAS-Model/ocean/develop`` (even though the
+We don't start off on ``E3SM-Project/E3SM/ocean/develop`` (even though the
```

Comment:
Is there still an `ocean/develop` branch in E3SM?  Maybe I haven't been paying close enough attention.

Reply:
```suggestion
We don't start off on ``E3SM-Project/E3SM/master`` (even though the
```